src/Backend/Modules/Location/Domain/Location.php

```diff
@@ -0,0 +1,295 @@
+<?php
+
+namespace Backend\Modules\Location\Domain;
+
+use Common\Locale;
+use Backend\Core\Language\Locale as BackendLocale;
+use Frontend\Core\Language\Locale as FrontendLocale;
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="location")
```

Comment:
you are going to switch from myissam to innodb here because that is the default in doctrine

Reply:
Shouldn't be an issue

Reply:
We're going to have to switch to InnoDB any way because we need relations (not now but in the future)